src/components/drawer/drawer.scss

```diff
@@ -0,0 +1,13 @@
+@import "@material/drawer/mdc-drawer";
+@import "@material/list/mdc-list";
+
+.drawer {
+  &__list-item {
+    padding: 0 15px 15px;
```

Comment:
please remove this extra padding -- we try to use default styles of material design as much as possible